packages/core/src/domain/configuration/configuration.ts

```diff
@@ -58,6 +59,12 @@ export interface InitConfiguration {
 // for this option.
 type GenericBeforeSendCallback = (event: any, context?: any) => unknown
 
+/**
+ * path: /api/vX/product
+ * parameters: xxx=yyy&zzz=aaa
+ */
+type ProxyFn = (path: string, parameters: string) => string
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏A single argument object with `path` and `parameters` properties could be more future-proof

```suggestion
type ProxyFn = (options: { path: string, parameters: string }) => string
```

Reply:
Do you have changes in mind that would be easier to handle with this approach?

Reply:
It could help if we wanted to have more fields or deprecate some...

Reply:
hum, looking at our current APIs, it seems that we are mostly using parameters.
We've moved from a parameter to an options object for [startView](https://github.com/DataDog/browser-sdk/pull/1448/files#diff-63186209ba3af0e1cccdc814069d547e89e9ff71e4dbc592b7cd210dbc4081ea) but from my understanding it was more for handling the fact that all parameters are optionals, which is not the case here.

I'd be in favour of being consistent in our API signatures, so if we want to move toward this pattern, it could be nice to give it a bit more thoughts.
Let's discuss it more.